src/ipvs/ip_vs_conn.c

```diff
@@ -1038,10 +1073,7 @@ void dp_vs_conn_put(struct dp_vs_conn *conn)
 /* used in conn timer handler: conn_expire */
 static void dp_vs_conn_put_nolock(struct dp_vs_conn *conn)
 {
-    if (conn->flags & DPVS_CONN_F_TEMPLATE)
-        dpvs_timer_update_nolock(&conn->timer, &conn->timeout, true);
-    else
-        dpvs_timer_update_nolock(&conn->timer, &conn->timeout, false);
+    dp_vs_conn_refresh_timer(conn);
```

Comment:
"-nolock" timer api should be used here.

Reply:
Done.